src/test/pages/customersPage.page.ts

```diff
@@ -70,14 +74,14 @@ export class CustomerPage extends MembersPage {
 
     public async assertAddedMembership() {
         await expect(this.currentMembershipLocator).toContainText(this.membershipPlanName);
-        await this.page.locator(".fa-xmark").click()
+        await this.xMark.click()
     }
 
-    public async checkAddedMembership() {
+    public async checkAddedMembership(customer:string) {
         //await expect(this.membersTab).toBeVisible();
         await this.membersTab.click();
-        await new Promise(resolve => setTimeout(resolve, 25000));
-        await expect(this.page.waitForSelector(`text=${this.customerName}`)).not.toBeNull();
+        await new Promise(resolve => setTimeout(resolve, 1000));
```

Comment:
same here with promises? Did you try to configure default timeout in config file?

Reply:
Yes i did, default timeout is configured both in config file and hooks file